### ISSUE TITLE
snap file

### ISSUE THREAD

**Issue by jiyoungAn** [JiahuiYu/generative_inpainting#50]
Hello. I have another question.
In each epoch, snap file is created and added the line in checkpoint.
`model_checkpoint_path: "snap-30000"
all_model_checkpoint_paths: "snap-10000"
all_model_checkpoint_paths: "snap-20000"
all_model_checkpoint_paths: "snap-30000"
`
What does it mean? each epoch has different loss value.
In your pretrained model, there is only one snap model.
Do I choose one snap with low loss value or 
Is it a structure that uses all snap files together?

Regarding, JiyoungAn

**Comment by JiahuiYu** [comment]
I chose the recent snap, renamed to snap-0.
I follow google's slim release template. See tensorflow / slim.
